scania-digital-design-system/tegel
# docs: document unnamed slots

**Describe pull-request**  
Adds doc comments for all unnamed slots.

**Solving issue**  
Fixes: #DTS-2098

**How to test**  
Because of the high number of changed files i suggest a smoke test: Select 5-10 stories randomly and only check them, that should be enough.

So:

1. Go to Storybook
2. Check in a ca 5-10 randomly selected stories, are unnamed slots documented?

**Suggested test steps**
